patientsearch/src/js/components/Search.js

```diff
@@ -140,24 +172,42 @@ async function fetchData(url) {
     }
 
     return json;
-  }
+}
 
+function Alert(props) {
+    return <MuiAlert elevation={6} variant="filled" {...props} />;
+}
+  
 export default function Search() {
     let focusInput = React.useRef(null);
     const classes = useStyles();
+    const [appReady, setAppReady] = React.useState(false);
+    const [appSettings, setAppSettings] = React.useState({});
     const [loading, setLoading] = React.useState(false);
     const [errorMessage, setErrorMessage] = React.useState("");
+    const [popOpen, setPop] = React.useState(false);
+    const [resultOpen, setResultOpen] = React.useState(false);
     const [firstName, setFirstName] = React.useState("");
     const [lastName, setLastName] = React.useState("");
     const [dob, setDOB] = React.useState(null);
     const [success, setSuccess] = React.useState(false);
+    const rootClass = clsx(classes.root, {
+        [classes.ready]: appReady,
+        [classes.notReady]: !appReady
+    });
     const buttonClassname = clsx({
         [classes.buttonSuccess]: success,
     });
+    const [searchResults, setSearchResults] = React.useState([]);
 
-    //TODO this is not a correct one!
-    const getLaunchURL = () => {
-        return "https://cosri-dev.cirg.washington.edu/auth/launch?launch=eyJhIjoiMSIsImIiOiI0MTcwMiIsImUiOiJTTUFSVC0xMjM0In0&iss=https%3A%2F%2Fsmart-dev-sandbox-launcher.cirg.washington.edu%2Fv%2Fr2%2Ffhir";
+    const getLaunchURL = (patientId) => {
+        if (!patientId) {
+            console.log("Missing information: patient Id");
+        }
+        let baseURL = appSettings["SOF_CLIENT_LAUNCH_URL"];
+        let iss = appSettings["SOF_HOST_FHIR_URL"];
+        let launchParam = btoa(JSON.stringify({"b":patientId}));
+        return `${baseURL}/auth/launch?launch=${launchParam}&iss=${iss}`; 
```

Comment:
```suggestion
        return `${baseURL}?launch=${launchParam}&iss=${iss}`; 
```
Going to make this minor change (move path into config), so we don't have to use the same view name everywhere